crypto/crypto.go

```diff
@@ -30,6 +30,15 @@ var S256 = btcec.S256()
 var (
 	secp256k1N = hex.MustDecodeHex("0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141")
 	one        = []byte{0x01}
+
+	ErrInvalidBLSSignature = errors.New("invalid BLS Signature")
```

Comment:
Literally smallest nitpick: 
Personally i prefer to have separete var for erros and not to mix it with other vars
var (
     Err1
     Err2
     Err3
)

this is just my personal opinion, if you don't agree i am ok with you resolving this comment :)